app.json

```diff
@@ -3,11 +3,8 @@
   "description": "A basic application using the GoCardless client to set-up customer mandates and subscriptions for the Fred Roche Foundation",
   "success_url": "/",
   "env": {
-    "GC_API_KEY_ID": {
```

Comment:
These were just old names. This file just gives a nice description of requirements, so it's fine that it was obsolete but better to keep it up to date.